sky/serve/controller.py

```diff
@@ -42,13 +44,17 @@ def __init__(self, service_name: str, service_spec: serve.SkyServiceSpec,
                  task_yaml: str, port: int) -> None:
         self._service_name = service_name
         self._replica_manager: replica_managers.ReplicaManager = (
-            replica_managers.SkyPilotReplicaManager(service_name=service_name,
-                                                    spec=service_spec,
-                                                    task_yaml_path=task_yaml))
+            replica_managers.SkyPilotReplicaManager(
+                service_name=service_name,
+                spec=service_spec,
+                initial_version=constants.INITIAL_VERSION,
+                task_yaml_path=task_yaml))
         self._autoscaler: autoscalers.Autoscaler = (
             autoscalers.RequestRateAutoscaler(
                 service_spec,
-                qps_window_size=constants.AUTOSCALER_QPS_WINDOW_SIZE_SECONDS))
+                qps_window_size=constants.AUTOSCALER_QPS_WINDOW_SIZE_SECONDS,
```

Comment:
Also, for this argument, seems we are passing the same constant all over the places. Do we really need it?

Reply:
In the future, we might want to expose this option to the user

Reply:
removed